src/db.rs

```diff
@@ -85,6 +141,83 @@ pub fn insert_connection(connection: Connection) -> Result<Uuid> {
     Ok(nwuuid)
 }
 
+pub fn insert_policy(policy: &policy::Policy) -> Result<i64> {
+    let mut dbconn = get_dbconn().unwrap();
+
+    let mysqlstr = "INSERT INTO policy (allowed_digests, allowed_policies, min_fw_api_major, min_fw_api_minor, 
+                    allowed_build_ids, create_date, valid)
+                    VALUES(?, ?, ?, ?, ?, NOW(), 1)".to_string();
+
+    let mut trnsx = dbconn.start_transaction(TxOpts::default())?;
+    /* create JSON for vector struct member variables */
+    let adj = serde_json::to_string(&policy.allowed_digests)?;
+    let apj = serde_json::to_string(&policy.allowed_policies)?;
+    let abidj = serde_json::to_string(&policy.allowed_build_ids)?;
```

Comment:
these variable names are slightly hard for me to understand

Reply:
I updated these with more descriptive snake case variables.